src/components/page/page.comp.tsx

```diff
@@ -450,6 +450,17 @@ const PageComp = ({ context }: IProps) => {
     );
   }
 
+  function generateItemsCountLabel() {
+    if (customLabels?.pagination?.itemsCount) {
+      const label = customLabels?.pagination?.itemsCount
+        .replace(':currentCount', JSON.stringify(items.length))
```

Comment:
Why do you need wrap it with `JSON.stringify`?

Reply:
The `replace` function from `Array`, although it is functional with a number as a second parameter, expects a `string` in its Typescript definition.

Reply:
If it's just a TypeScript thing, maybe instead we could use `as string`?